app/Http/Controllers/CompanyController.php

```diff
@@ -107,9 +109,14 @@ public function update(Request $request)
 
     public function destroy(string $id)
     {
-        $company = Company::find($id);
-        $projects = $company->projects()->count();
-        Company::destroy($id);
-        return Response::json( "The Company " . $company->name . ", and ". $projects ." related project records Were Deleted");
+        try {
+            $company = Company::find($id);
+            $projects = $company->projects()->count();
+            $msg = "The Company " . $company->name . ", and " . $projects . " related project records Were Deleted";
```

Comment:
why are you passing a variable instead of making a direct response?

Reply:
cant access the `company->name` after its destroyed therefore....im using this workaround @Raza9798

Reply:
it can be accessed directly try a different approach.

Reply:
Okay Sure...